### DISCUSSION TITLE
App extends across status bar after upgrading to 5.5

### DISCUSSION THREAD

**Discussion by Zerachiel01** [unoplatform/uno#19028] [Q&A] [answered]
Hello everyone!

So I upgraded my Uno Platform Android App from 5.2 to 5.5 and my app suddenly extends across the status bar at the top. In 5.2, my App-UI "started" below the status bar, now it overlaps with the status bar (I hope I can describe this properly).

Is there a way to get the old behaviour back? As of now, I want my app to start below the status bar.

Thanks in advance :)

**Comment by morning4coffe-dev** [comment, marked as answer]
Hey @Zerachiel01,

are you using [SafeArea](https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/controls/SafeArea.html) in your app? If so, could you create a small repro for me, please? I’m not seeing the issue on my side. Thanks!

**Reply by jeromelaban** [reply]
Also, can you provide a repro of this behavior? Make sure to specify the environment you're running the app on as well.

**Reply by Zerachiel01** [reply]
Hello,
thanks for the quick answer. I was actually not using SafeArea, I did not know about it and seem to have removed it from my blank app long time ago.
Using SafeArea solves my issue, thank you very much!

I do believe that something changed between 5.2 and 5.5 though. I ran an old sample app of mine that is still on 5.2 and removed all the uses of SafeArea and it seems to still keep everything in the visible bounds as if it uses SafeArea-VisibleBounds anyway. Maybe SafeArea did not work properly in 5.2 and got fixed in 5.3+? That would explain why I realized the missing SafeArea in my app just now when I upgraded to 5.5.
Maybe this is something you guys could give a try?
The environment I am running on is a Google Pixel 7a with Android 15 installed. (not emulated)
